.gitignore

```diff
@@ -23,3 +23,6 @@
 /try_ruby.rb
 /coverage/*
 /vendor/bundle/*
+
+# Ignore application configuration
+# /config/application.yml
```

Comment:
i know that I am not allowed to push this file, but still if you will try to run app, you'll get error